config/jobs/kubernetes/sig-cluster-lifecycle/kubeadm-kinder.yaml

```diff
@@ -46,7 +46,7 @@ periodics:
           cpu: 2000m
 
 - name: ci-kubernetes-e2e-kubeadm-kinder-1-27
-  cluster: eks-prow-build-cluster
+  cluster: k8s-infra-prow-build
   interval: 12h
```

Comment:
this is expected.
sig k8s infra folks wanted to have one kubeadm job on the eks cluster, but our tools override it.

perhaps one day we can switch to eks for all jobs

Reply:
Reverted.